src/algorithms/reco/ScatteredElectronsEMinusPz.h

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2024 Daniel Brandenburg
+
+#pragma once
+
+#include <edm4eic/ReconstructedParticleCollection.h>
+#include <spdlog/logger.h>
+#include <memory>
+
+#include "ScatteredElectronsEMinusPzConfig.h"
+#include "algorithms/interfaces/WithPodConfig.h"
+
+
+namespace eicrecon {
+
+    class ScatteredElectronsEMinusPz : public WithPodConfig<ScatteredElectronsEMinusPzConfig>{
```

Comment:
ScatteredElectronsEMinusPz should be converted to use the `algorithms::` interface.

This would bring this in line with what ScatteredElectronsTruth already does, and simplify the interface. This can be out of scope for this PR, but a separate issue to be created from this comment and tracked.

Reply:
See #1360.